src/schemas/constants.ts

```diff
@@ -7,8 +7,12 @@ export const MESSAGES = {
   PASSWORD_REPEAT_REQUIRED: 'Please confirm your password',
   PASSWORD_MIN: 'Password must be at least 8 characters long',
   PASSWORD_MAX: 'Password must be no longer than 20 characters',
+  COMMENT_REQUIRED: 'Comment is required',
+  COMMENT_MIN: 'Comment must be at least 7 characters long',
+  COMMENT_MAX: 'Comment must be no longer than 230 characters',
```

Comment:
Why we put constants in schemas folder? Better make constants file or folder separately